src/modes/OptimInterface.jl

```diff
@@ -7,31 +7,32 @@ import ..ForwardDiff
 import NamedArrays
 import StatsBase
 import Printf
+import StatsAPI
```

Comment:
Added the import

Reply:
You also have to add StatsAPI to the list of dependencies (in Project.toml), together with a compat entry in the `[compat]` section.

Reply:
I would recommend reading through the Pkg docs regarding how to manage packages and their dependencies (in particular sections 2 to 4): https://pkgdocs.julialang.org/v1/ I also recommend working with project environments whenever working with Julia.

Reply:
But yeah, we can do it for you, np :+1: